styleguide/samples/_forms.html.haml

```diff
@@ -0,0 +1,53 @@
+        %h1 Consumer Query
+        %p From 1 April 2019 we are only able to provide general consumer advice to residents of England and Wales. If you are resident in Scotland and have a general consumer enquiry please contact Advice Direct Scotland / 0808 164 6000 
+        
+        = render partial: "@citizensadvice/design-system/haml/input", locals: { input: { "name" => "example-input", "label" => "First name or initials" }}
+
+        = render partial: "@citizensadvice/design-system/haml/input", locals: { input: { "name" => "example-input", "label" => "Last name" }}
+
+        = render partial: "@citizensadvice/design-system/haml/textarea", locals: { input: { "name" => "example-textarea", "label" => "Your complaint or enquiry", "hint" => "Please provide details of your complaint or enquiry." }}
+
+        %h2 Purchase details
+
+        = render partial: "@citizensadvice/design-system/haml/radio_group", locals: { radio: { "name" => "example-radio-group", "label" => "Have you paid for goods or services from this trader?", "options" => [{ "value" => "opt1", "label" => "Yes - If yes, answer the below questions" }, { "value" => "opt2", "label" => " No" }] } }
+
+        = render partial: "@citizensadvice/design-system/haml/input", locals: { input: { "name" => "example-input", "label" => "What was the total amount paid for the goods or services?", "hint" => "Enter a value in pounds and pence only e.g. £112.00" }}
+
+        = render partial: "@citizensadvice/design-system/haml/radio_group", locals: { radio: { "name" => "example-radio-group", "label" => "Have you contacted the trader about this complaint? ", "options" => [{ "value" => "opt1", "label" => "Yes - If yes, answer the below questions" }, { "value" => "opt2", "label" => " No" }] } }
+
+        = render partial: "@citizensadvice/design-system/haml/textarea", locals: { input: { "name" => "example-textarea", "label" => "Outline the trader's response to the complaint, if any"}}
+
+        `<p><button type="button" class="cads-button cads-button__primary">Submit complaint</button></p>`
+
+      .cads-grid-col-md-4
+        %aside.cads-related-content
+          %p.paragraph-s.section-title Related Content
+          %h3
+            %a{:href=> 'https://www.citizensadvice.org.uk'} Search and navigation tools
+          %ul
+            %li
+              %a{:href=> 'https://www.citizensadvice.org.uk'} Search
+            %li
+              %a{:href=> 'https://www.citizensadvice.org.uk'} A to Z of advice
+            %li
+              %a{:href=> 'https://www.citizensadvice.org.uk'} Finding information
+            %li
+              %a{:href=> 'https://www.citizensadvice.org.uk'} Search for your local Citizens Advice
+            %li
+              %a{:href=> 'https://www.citizensadvice.org.uk'} Browse our campaign pages
+            %li
+              %a{:href=> 'https://www.citizensadvice.org.uk'} Browse our research pages
+= render partial: "@citizensadvice/design-system/haml/footer"
```

Comment:
Sadly the build failed for a pedantic reason. It wants you to put an empty new line at the end of this file